Nucleation_model_radial_average.py

```diff
@@ -34,8 +34,7 @@
 
 C_LiO2_sat = 0.1 #mol m-3 // saturated concentration of LiO2 //Yin (2017)
 C_Li_sat = 0.1 #mol m-3 // saturated concentration of LiO2
-k_nu = 1E-6 #mol s-1 m-2 // kinetic rate constant for the nucleation reaction // Yin (2017)
-k_surf = 1E-8 #mol s-1 m-2 // kinetic rate constant for the growth reaction // Yin (2017)
+k_surf = 1E-15 #mol s-1 m-2 // kinetic rate constant for the growth reaction // Yin (2017)
 k_surf_des = 1E-8 #mol s-1 m-2 // kinetic rate constant for the growth reaction // Yin (2017)
```

Comment:
Your comments for `k_surf` and `k_surf_des` are the same.  Can you modify to say how they are different?  Is one of them for the nucleation reaction?